tgui/packages/tgui/interfaces/Biogenerator.js

```diff
@@ -0,0 +1,190 @@
+import { classes } from 'common/react';
+import { createSearch } from 'common/string';
+import { Fragment } from 'inferno';
+import { useBackend, useLocalState } from '../backend';
+import { Box, Button, Dimmer, Flex, Icon, Input, Section, Table, Tabs, NoticeBox, NumberInput } from '../components';
+import { formatMoney } from '../format';
+import { Window } from '../layouts';
+
+const MAX_SEARCH_RESULTS = 25;
+
+export const Biogenerator = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    beaker,
+    processing,
+  } = data;
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        {!beaker && (
+          <NoticeBox>No Container</NoticeBox>
+        )}
+        {!!processing && (
+          <Dimmer fontSize="32px">
```

Comment:
Pull Dimmer outside of window content to make it not scroll with it.

Reply:
Done.